packages/os/os.spec

```diff
@@ -147,6 +147,13 @@ Summary: Thar data store migrations
 %description -n %{_cross_os}migrations
 %{summary}.
 
+%if "%{_cross_variant}" == "aws-ecs-1"
+%package -n %{_cross_os}ecs-settings-applier
```

Comment:
Can this only be built on the aws-ecs-1 variant due to a dependency on ECS-specific models?

Reply:
Correct